app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+import shutil
+
+from app.validator import validation_command
+
+
+def move_file(command: str) -> None:
+    import shlex
```

Comment:
Don't use `import` inside function. You should place all your imports on start of file